ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.12.yaml

```diff
@@ -222,6 +252,34 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-upi
+- as: azure-ipi-disc-fullyprivate-f28
+  cron: 37 9 13 * *
+  steps:
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:arm64-target
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-upgrade-qe-test-disconnected
+    workflow: cucushift-installer-rehearse-azure-ipi-disconnected-fullyprivate
+- as: azure-ipi-ultrassd-disks-f28
```

Comment:
This job is not placed in alphabet order.

Reply:
Isn't the other of sorting `...r-s-t-u-v...` , so:
azure-ipi-resourcegroup-f28
azure-ipi-ultrassd-disks-fips-f28
azure-ipi-vmgenv1-f28

Reply:
The previous job is `azure-ipi-disc-fullyprivate`, the next job is `azure-ipi-fullyprivate-proxy`, right?